contracts/ForkDelta.sol

```diff
@@ -58,12 +56,6 @@ contract ForkDelta {
     feeAccount = feeAccount_;
   }
 
-  /// Changes the fee on makes. Can only be changed to a value less than it is currently set at.
```

Comment:
While this is also a changed signature, it was an administrative one that the ForkDelta API doesn't directly interact with